catalogue/webapp/pages/images.tsx

```diff
@@ -169,7 +170,7 @@ const Images: NextPage<Props> = ({
           className={classNames(['row'])}
         >
           <div className="container">
-            {!results && <SearchTitle />}
+            {!isCollectionsPage && <SearchTitle />}
```

Comment:
using !results seems better to me - doesn't isCollectionPage just mean it has results - I find the name a bit confusing

Reply:
sounds good , will do